Sources/PackageModel/UserToolchain.swift

```diff
@@ -382,11 +382,11 @@ public final class UserToolchain: Toolchain {
                             // the merged swiftmodules.  XCTest followed suit.
                             "-I",
                             AbsolutePath(
-                                validating: "usr/lib/swift/windows/\(triple.arch)",
+                                validating: "usr/lib/swift/windows/\(triple.arch?.rawValue ?? "unknown")",
```

Comment:
I'm somewhat uncomfortable with this.  The `unknown` here is confusing.  Would we be able to replace this with something like `<<NULL>>` or something that is explicitly bad and obviously so?

Reply:
should this throw in case it is null?

Reply:
I think that having it throw would be a much safer option.  We need to know the architecture to be able to build.

Reply:
Resolved this by using `archName` which is non-optional